language/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -769,3 +770,22 @@ impl ResolvedPackage {
         }
     }
 }
+
+impl ResolvedGraph {
+    pub fn file_sources(&self) -> BTreeMap<FileHash, (Symbol, String)> {
+        self.package_table
+            .iter()
+            .flat_map(|(_, rpkg)| {
+                rpkg.get_sources(&self.build_options)
+                    .unwrap()
+                    .iter()
+                    .map(|fname| {
+                        let contents = read_to_string(Path::new(fname.as_str())).unwrap();
```

Comment:
nit, bit more standard to `use std::fs` and then `fs::read_to_string`, but not a big deal either way